airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/JdbcSqlOperations.java

```diff
@@ -137,7 +128,8 @@ public final void insertRecords(final JdbcDatabase database,
       throws Exception {
     AirbyteSentry.executeWithTracing("InsertRecords",
         () -> {
-          records.forEach(airbyteRecordMessage -> getDataAdapter().adapt(airbyteRecordMessage.getData()));
```

Comment:
This is bad. For every record message, a new data adapter object is created (in the `getDataAdapter` method).

Reply:
obviously this is already way better, but could we just dump the data adapter into a field?

Reply:
Ah, good point! Done.